anvio/workflows/metagenomics/Snakefile

```diff
@@ -783,7 +783,7 @@ if sample_name != M.default_config['anvi_profile']['--sample-name'] and sample_n
 
 def get_cluster_contigs_param(wildcards):
     """ helper function to sort out whether to cluster contigs for a single profile database"""
-    if M.get_param_value_from_config(['anvi_profile', '--cluster-contigs'] is not None):
+    if M.get_param_value_from_config(['anvi_profile', '--cluster-contigs']) is not None:
```

Comment:
😂

Reply:
yeah, it is embarrassing 😂

Reply:
No bug that is caught is embarrassing! Embarrassing bugs are those that linger for ages (at which I excel).